server/server.js

```diff
@@ -110,12 +136,33 @@ var upload = multer({
  */
 var exonFile = dataPath + 'exons.bin';
 
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(bodyParser.json());
+/**
+ * User authenticate handler.
+ */
+MongoClient.connect(mongoUrl, function(err, db) {
+  if (err) {
+    log.serverLog(err.message);
+    return;
+  }
+  log.serverLog('connected to MongoDB');
+  database.db = db;
+
+  // Start the application.
+  //var privateKey = fs.readFileSync(privateKeyPath);
+  //var certificate = fs.readFileSync(certificatePath);
+  //var httpsServer = https.createServer({
+  //  key: privateKey,
+  //  cert: certificate
+  //}, app).listen(443);
+  //httpsServer.setTimeout(1200000);
+
+  var server = app.listen(3000);
+  server.setTimeout(1200000);
+});
 
 var serverResponse = function(data, res) {
```

Comment:
Okay. Then it is not necessary to res.json(data). You can just reply with a 200 header.